managing_providers/_topics/containers_providers.md

```diff
@@ -55,6 +55,32 @@ When deploying OpenShift using `openshift-ansible-3.0.20` (or later
 versions), the OpenShift Container Platform [service account](https://docs.openshift.com/container-platform/4.5/authentication/understanding-and-creating-service-accounts.html)
 and [roles](https://docs.openshift.com/container-platform/4.5/authentication/understanding-and-creating-service-accounts.html#service-accounts-granting-roles_understanding-service-accounts) required by {{ site.data.product.title_short }} are installed by default.
 
+For newer versions of OpenShift you have to create a service-account with the proper permissions for {{ site.data.product.title_short }}.
+
+1. Create a namespace for the service account
+```
```

Comment:
Indent these blocks with 3 spaces (so lined up with the text) - this will render the code block "inside" the bullet list, as opposed to outside it.